test_runner/regress/test_tenants.py

```diff
@@ -397,3 +398,30 @@ def test_pageserver_with_empty_tenants(
     assert (
         tenant_broken_count == 1
     ), f"Tenant {tenant_without_timelines_dir} should have metric as broken"
+
+
+# Check that empty tenants work with or without the remote storage
+@pytest.mark.parametrize(
+    "remote_storage_kind", available_remote_storages() + [RemoteStorageKind.NOOP]
```

Comment:
I wonder if we really need more than LocalFS for this. What's the rationale for NOOP?

Though I wonder if there should be anything observable for remote storages. Probably not, because we don't upload anything before a timeline is created, so no suggestions on asserts. I just think we should be mindful of the real_s3 test cases we create.